src/app/visual/readonly-grid/readonly-grid.component.ts

```diff
@@ -61,39 +61,82 @@ export class ReadonlyGridComponent implements OnInit {
     {
       field: 'comment',
       headerName: 'Comment',
-      minWidth: 400
+      maxWidth: 500,
+      autoHeight: true,
+      cellRenderer: (params: ICellRendererParams) => {
+        return `<div style="white-space: normal">${params.value || ''}</div>`;
+      }
     },
     {
       field: 'status',
       headerName: 'Status',
       sortable: false,
-      cellRenderer: this.statusRenderer
+      cellRenderer: this.statusRenderer,
+      minWidth: 300
     }];
 
   constructor(private agGridService: SkyAgGridService) { }
 
   public ngOnInit(): void {
     this.gridOptions = {
       columnDefs: this.columnDefs,
-      onGridReady: gridReadyEvent => this.onGridReady(gridReadyEvent)
+      onGridReady: gridReadyEvent => this.onGridReady(gridReadyEvent),
+      domLayout: 'autoHeight',
+      alignedGrids: []
     };
     this.gridOptions = this.agGridService.getGridOptions({ gridOptions: this.gridOptions });
   }
 
+  public onScrollEnd(): void {
+    if (this.hasMore) {
+      // MAKE API REQUEST HERE
+      // I am faking an API request because I don't have one to work with
+      this.mockRemote().then((result: any) => {
+        this.gridData = this.gridData.concat(result.data);
+        this.hasMore = result.hasMore;
+      });
+    }
+  }
+
+  public mockRemote(): Promise<any> {
```

Comment:
Should this use an `Observable` instead, since that's what consumers will most likely be using when retrieving remote data?